### PR TITLE
Added /fireball and /nuke commands

### PR DESCRIPTION
**Pull request by mathias-github**

### CODE REVIEW

**Comment by tonibm19** [comment]
Great

File: cheat.lua [hunk 1]
```diff
@@ -290,3 +290,47 @@ function HandlePowertoolCommand(Split, Player)
...
+	end
+end
+
+function HandleNukeCommand(Split, Player)
+	local SpawnNuke = function(OtherPlayer)
+		local X = OtherPlayer:GetPosX()
+		local Y = OtherPlayer:GetPosY() + 35
+		local Z = OtherPlayer:GetPosZ()
+		OtherPlayer:SendMessageInfo("May death rain upon them")
+		OtherPlayer:GetWorld():SpawnPrimedTNT(X, Y, Z, 52)
```
Comment (NiLSPACE) [inline cheat.lua]: I'm a little late, but wouldn't this be allot nicer to do like this?:
```lua
for x = -2, 2, 2 do
   for z = -2, 2, 2 do
      OtherPlayer:GetWorld():SpawnPrimedTNT(X + x, Y, Z + z, 52)
   end
end
```

Comment (mathiascode) [inline cheat.lua]: Definitely. 

The command seems to cause server crashes more often than usual. I need to look into that too.

Comment (NiLSPACE) [inline cheat.lua]: It's very possible that the TNT entity is just rather unstable which is causing the crashes.

**Pull request merged**